sentences/de/climate_HassClimateGetTemperature.yaml

```diff
@@ -7,4 +7,5 @@ intents:
           - "Wie hoch ist [<area>] die Temperatur"
           - "Wie hoch ist die Temperatur (von | vom) <name>"
           - "Wie hoch ist die Temperatur vom Thermostat <area>"
-          - "Wie warm ist es <area>"
\ No newline at end of file
+          - "Wie warm ist es <area>"
+          - "Wie viel Grad {temperature_unit} hat es [<area>]"
```

Comment:
this is already be added with #445. but your tests are a nice addition, so would be great if you would contribute the tests into #445
thx 👍

Reply:
Added the test sentences to #445